python-sdk/tests/databases/test_snowflake.py

```diff
@@ -165,3 +165,58 @@ def test_snowflake_load_options_empty():
     assert not load_options.empty()
     load_options.file_options = {}
     assert load_options.empty()
+
+
+def test_storage_integrations_params():
+    """
+    test that if `storage_integration` is not passed to `load_file_to_table_natively` method either via
+    `native_support_kwargs` or via `SnowflakeLoadOptions` it raises an exception.
+    """
+    table = Table(conn_id="fake-conn")
+    file = File(path="azure://data/homes_main.ndjson")
+    database = SnowflakeDatabase(conn_id=table.conn_id, load_options=SnowflakeLoadOptions())
+    with pytest.raises(DatabaseCustomError) as e:
+        database.load_file_to_table_natively(source_file=file, target_table=table)
+    assert e.match(
+        "Param 'storage_integration' is required. Please pass either in `native_support_kwargs` or"
+        " `SnowflakeLoadOptions.copy_options`"
```

Comment:
this should be `SnowflakeLoadOptions.storage_integration` or `SnowflakeLoadOptions.copy_options`?

Reply:
Yup, will change this. Thanks for pointing out